Privacy-policy-snippets.md

```diff
@@ -0,0 +1,36 @@
+# Privacy Policy Snippets
+
+Cut and paste-able snippets that describe how core collects, manages and shares data provided by site users and visitors.
+
+## Definitions
+
+* Logged In vs Logged Out Users: Logged in users are users who have registered (created) an account on the site and have provided their username and password to log in to the site. Some sites require users to be registered and logged in to do things like comment on a post. Logged out users are users who have not logged in, or who do not have an account on the site
+at all (e.g. visitors)
+* Gravatar: An optional public profile, including a photo or image, related to your email address.
+* IP Address: The address of your device on the Internet.  This address is assigned by the Internet Service Provider providing the connection to the Internet which your device is using and may be shared with other devices at that same location.
+* Role: A logged in user also has a role, e.g. Contributor, Author, Editor, etc. Each role has a prescribed set of capabilities - e.g. some roles can delete comments but others cannot.
+* User Agent: A browser’s user agent is a line of text that usually includes which browser you are using, its version, and your operating system and its version.
+
+## Comments
+
+### Comments Left by All Users
+
+* If you elect to leave a comment on a post, in addition to the comment text you provide we also collect your IP Address and your User Agent.
+* Your comment text, your name, email address, website URL, IP Address and User Agent are accessible by administrators on our site.
+* A hash of your email address is provided to the Gravatar service to see if a profile picture of you is available for display. The Gravatar service privacy policy is available here: https://automattic.com/privacy/ Prior to approval of your comment, your profile picture is only visible to administrators. After approval of your comment, your profile picture is visible to the public in the context of your comment.
+* Following approval of your comment, your comment text, your name, and website URL (if provided) are visible to the public.
+* Comments containing links or certain words or phrases may require manual approval by an administrator
```

Comment:
Missing period on end of sentence.

Reply:
Fixed in https://github.com/gdpr-compliance/info/pull/8/commits/3481f3b64f5138e52a3fbfc2c06c6d6c9c14859b